_posts/en/newsletters/2023-11-08-newsletter.md

```diff
@@ -0,0 +1,249 @@
+      allows fee estimates to be updated asynchronously (in a different thread)
+      so that validation and relay can complete more quickly.
+      It may also make testing `CTxMempool` easier.
+      Finally, it allows the future use of more complex fee estimation
+      algorithms without affecting block validation and relay performance."
+  a0link="https://bitcoincore.reviews/28368#l-30"
+
+  q1="Isn't fee estimation currently updated synchronously when transactions
+      are added to or removed from the mempool even without a new block arriving?"
+  a1="Yes, but performance isn't as critical at those times as during
+      block validation and relay."
+  a1link="https://bitcoincore.reviews/28368#l-41"
+
+  q2="Are there any benefits of the `CBlockPolicyEstimator` being a member
+      of `CTxMempool` and updating it synchronously (the current arrangement)?
+      Are there downsides to removing it?"
+  a2="Synchronous code is simpler and easier to reason about. Also, the fee
+      estimator has more visibility into the entire mempool; a downside is
+      the need to encapsulate all the information needed for fee estimation
+      into a new `NewMempoolTransactionInfo` structure.
+      However, not much information is needed by the fee estimator."
+  a2link="https://bitcoincore.reviews/28368#l-43"
+
+  q3="What do you think are the advantages and disadvantages of the approach
+      taken in this PR, compared with the one taken in [PR 11775][] that
+      splits `CValidationInterface`?"
+  a3="While it seems nice to split them, they still had a shared backend
+      (to keep the events well-ordered), so weren't really very independent
+      of each other.
+      There doesn't seem to be much practical benefit to splitting.
+      The current PR is more narrow, and minimally scoped to making fee
+      estimates update asynchronously."
+  a3link="https://bitcoincore.reviews/28368#l-71"
+
+  q4="In a subclass, why is implementing a `CValidationInterface` method
+      equivalent to subscribing to the event?"
+  a4="All subclasses of  `CValidationInterface` are clients.
+      The subclass can implement some or all `CValidationInterface` methods (callbacks),
+      for example, connecting or disconnecting a block, or a transaction
+      [added][tx add] to or [removed][tx remove] from the mempool.
+      After being registered (by calling `RegisterSharedValidationInterface()`),
+      any implemented `CValidationInterface` method will be executed each time
+      the method callback is fired using `CMainSignals`.
+      Callbacks are fired whenever the corresponding event occurs."
+  a4link="https://bitcoincore.reviews/28368#l-90"
+
+  q5="[`BlockConnected`][BlockConnected] and [`NewPoWValidBlock`][NewPoWValidBlock]
+      are different callbacks.
+      Which one is asynchronous and which one is synchronous? How can you tell?"
+  a5="`BlockConnected` is asynchronous; `NewPoWValidBlock` is asynchronous.
```

Comment:
Isn't `NewPoWValidBlock` synchronous rather than asynchronous?

Reply:
Took this suggestion